lib/enum/enum.rb

```diff
@@ -40,8 +53,6 @@ def set_constant(record)
         end
       end
 
-      private :set_constant
```

Comment:
Just a note that this change is fine and the methods in question are still private because of the `private` declaration above. That said, I would very much prefer in the future that changes like this are presented in a separate refactoring commit.

Reply:
Fix: https://github.com/active-hash/active_hash/pull/321/commits/9e2fc62b37e5f108ed06ba2e7cd25fed40df7fb5